index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./index.css">
+    <link rel="shortcut icon" href="#" type="image/x-icon">
+    <title>Whack A Mole</title>
+</head>
+<body>
+<h1 class="title"> Whack A Mole
+    <span id="score">0</span>
+    <button id="button">Start Game</button>
+</h1>
```

Comment:
Why have you decided to include button and score in the h1 tag? Does it make sense?